recipes/wxwidgets/all/conanfile.py

```diff
@@ -180,6 +180,8 @@ def _patch_sources(self):
         # Fix for strcpy_s (fix upstream?)
         if is_apple_os(self):
```

Comment:
This fix is present in all available versions of the recipe, we might want to remove it